[2022-12-02 21:30:54,309] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2022-12-02T00:00:00+00:00 [queued]>
[2022-12-02 21:30:54,316] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2022-12-02T00:00:00+00:00 [queued]>
[2022-12-02 21:30:54,328] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-02 21:30:54,328] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-02 21:30:54,329] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-02 21:30:54,367] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2022-12-02 00:00:00+00:00
[2022-12-02 21:30:54,396] {standard_task_runner.py:52} INFO - Started process 61992 to run task
[2022-12-02 21:30:54,438] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'meu_primeiro_dag', 'cria_pasta', 'scheduled__2022-12-02T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/meu_primeiro_dag.py', '--cfg-path', '/tmp/tmp2i066dce', '--error-file', '/tmp/tmplorqrvb1']
[2022-12-02 21:30:54,440] {standard_task_runner.py:80} INFO - Job 24: Subtask cria_pasta
[2022-12-02 21:30:54,677] {task_command.py:370} INFO - Running <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2022-12-02T00:00:00+00:00 [running]> on host caiomendes-virtual-machine
[2022-12-02 21:30:54,755] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=meu_primeiro_dag
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2022-12-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-02T00:00:00+00:00
[2022-12-02 21:30:54,756] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-12-02 21:30:54,757] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/caiomendes/Documents/airflow/pasta=2022-12-03T00:00:00+00:00" ']
[2022-12-02 21:30:54,813] {subprocess.py:85} INFO - Output:
[2022-12-02 21:30:54,861] {subprocess.py:96} INFO - Command exited with return code 0
[2022-12-02 21:30:54,902] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=meu_primeiro_dag, task_id=cria_pasta, execution_date=20221202T000000, start_date=20221203T003054, end_date=20221203T003054
[2022-12-02 21:30:54,973] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-02 21:30:55,041] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
